### ISSUE TITLE
Any motivation to make this into an npm package? 

### ISSUE THREAD

**Issue by dkaushik95** [Kaljurand/dictate.js#21]
Hey,

Is there any motivation to make this into an npm package? I am trying to make an application where I will be using dictate.js with react. Are you motivated to make this into an npm package? I don't have any experience in building an npm package but I can help remove jquery and add npm-based javascript. 

Thanks,

**Comment by Kaljurand** [comment]
This might be a good idea, but I don't have any experience with npm either. Feel free to do this in a fork. As far as I can see there is no jQuery in dictate.js.

**Comment by dkaushik95** [comment]
There is a $ajax [here](https://github.com/Kaljurand/dictate.js/blob/master/lib/dictate.js#L190). I have made some changes to make it into a react component. I'll put a PR soon.

**Comment by Kaljurand** [comment]
Right, I wasn't even aware of that "$.ajax" call. I'd say the "submitReference" is not such an essential feature of "dictate.js", i.e. you might not need it for your use case. Maybe we could also remove it from "dictate.js" to have less dependencies. Adding @alumae for possible comments.
